pages/belt_docs.mdx

```diff
@@ -3,33 +3,114 @@ export default Layout;
 
 # Belt
 
-A stdlib shipped with BuckleScript This stdlib is still in beta but we
-encourage you to try it out and give us feedback.
+The preferred standard library for Reason when targeting JavaScript
+(browser, node, React Native).
+
+It is recommended to use the modules from Belt rather than from the
+OCaml standard library or from the Js namespace, e.g. use `Belt.List` instead of
+`List` or `Js.List` and `Belt.Array` instead of `Array` or `Js.Array`.
 
 ## Motivation
 
-The motivation for creating such library is to provide BuckleScript users a
-better end-to-end user experience, since the original OCaml stdlib was not
-written with JS in mind. Below is a list of areas this lib aims to improve:
+Belt improves over the OCaml standard library as follows:
+
+- Consistent naming convention familiar to JS developers:
+  - camlCase
+  - arguments order (see section "Pipe First" below)  
+- Exceptions:
+  - Only functions suffixed with "Exn" will throw exceptions.
+  - Other functions with an optional result will just return `option('a)`.
+- Better performance and smaller code size running on the JS platform
+
```

Comment:
There's also:
- performance
- tree-shaking-readty

Reply:
Thanks, will add that!